tools/run_tests/helper_scripts/build_python.sh

```diff
@@ -165,7 +165,7 @@ pip_install_dir_and_deps() {
 pip_install -U gevent
 
 pip_install --upgrade cython
-pip_install --upgrade six protobuf>=4.21.3
+pip_install --upgrade six protobuf>=4.21.3 typeguard
```

Comment:
Likewise, since `typeguard` is only a dependency of a Bazel-run test, we should not be installing it at the system level here.